astrology.lic

```diff
@@ -276,13 +289,14 @@ class Astrology
     fix_standing
   end
 
-  def train_astrology
-    check_ripples
+  def train_astrology(settings)
+    return if DRSkill.getxp('Astrology') > 30
```

Comment:
Going to add in an `astrology_training` list similar to what was done with `appraisal_training`